crates/liboci-cli/src/features.rs

```diff
@@ -6,4 +6,61 @@ use clap::Parser;
 /// It is documented here:
 /// https://github.com/opencontainers/runtime-spec/blob/main/features-linux.md
 #[derive(Parser, Debug)]
-pub struct Features {}
+pub struct Features {
+    /// Minimum OCI Runtime Spec version recognized by the runtime, e.g., "1.0.0".
+    oci_version_min: Option<String>,
+    /// Maximum OCI Runtime Spec version recognized by the runtime, e.g., "1.0.2-dev".
+    oci_version_max: Option<String>,
+    /// List of the recognized hook names, e.g., "createRuntime".
+    hooks: Option<Vec<String>>,
+    /// List of the recognized mount options, e.g., "ro".
+    mount_options: Option<Vec<String>>,
+    /// Specific to Linux.
+    linux: Option<Linux>,
+    /// Contains implementation-specific annotation strings.
+    annotations: Option<std::collections::HashMap<String, String>>,
+}
+
+/// Specific to Linux.
+#[derive(Debug, Clone)]
+pub struct Linux {
+    /// List of the recognized namespaces, e.g., "mount".
+    namespaces: Option<Vec<String>>,
+    /// List of the recognized capabilities, e.g., "CAP_SYS_ADMIN".
+    capabilities: Option<Vec<String>>,
+    cgroup: Option<Cgroup>,
+    seccomp: Option<Seccomp>,
+    apparmor: Option<Apparmor>,
+    selinux: Option<Selinux>,
+}
+
+#[derive(Debug)]
+struct Seccomp {
+    enabled: Option<bool>,
+    actions: Option<Vec<String>>,
+    operators: Option<Vec<String>>,
+    archs: Option<Vec<String>>,
+}
+
+#[derive(Debug)]
+struct Apparmor {
+    enabled: Option<bool>,
+}
+
+#[derive(Debug)]
+struct Selinux {
+    enabled: Option<bool>,
+}
+
+#[derive(Debug)]
+struct Cgroup {
+    v1: Option<bool>,
+    v2: Option<bool>,
+    systemd: Option<bool>,
+    systemd_user: Option<bool>,
+}
+
+pub const ANNOTATION_RUNC_VERSION: &str = "org.opencontainers.runc.version";
+pub const ANNOTATION_RUNC_COMMIT: &str = "org.opencontainers.runc.commit";
+pub const ANNOTATION_RUNC_CHECKPOINT_ENABLED: &str = "org.opencontainers.runc.checkpoint.enabled";
+pub const ANNOTATION_LIBSECCOMP_VERSION: &str = "io.github.seccomp.libseccomp.version";
```

Comment:
This is a matter of preference, but please define all constants at the top, after the `use` statements.